bin/run_atcontroller.sh

```diff
@@ -4,25 +4,16 @@ source $loc/base_init.sh
 
 if [ -z $1 ];
 then
-    echo "$0: missing argument: lsst_dds_partition_prefix container_version"
+    echo "$0: missing argument: container_version"
     exit 1
 fi
 
-if [ -z $2 ];
-then
-    echo "$0: missing argument: lsst_dds_partition_prefix container_version"
-    exit 1
-fi
-
-lsst_dds_partition_prefix=$1
 container_version=$2
```

Comment:
To my understanding, after the changes, script's first and only positional argument is the container version, right? If so, shouldn't the line 11 read
```bash
container_version=$1
```

Reply:
Yes.  I'll address this.